src/desktop_develop.js

```diff
@@ -25,7 +25,9 @@ const logger = require('./logger');
 const Runner = require('./runner');
 const DockerRunner = require('./docker_runner');
 
-const TYPES = ['mac', 'linux'];
+const WindowsBuilder = require('./windows_builder');
+
+const TYPES = ['win64', 'mac', 'linux'];
```

Comment:
Should they all say `64` then...? Seems a bit odd for only one to have the architectural size stated... Just a feeling though, feel free to ignore.

Reply:
32 bit on mac nowadays is old enough that 'mac64' sounds a bit weird - given they're only internal identifiers I'm minded to just leave the 32/64 bit suffixes on windows.